index.js

```diff
@@ -154,14 +164,14 @@ const recastNewUsers = async () => {
     if (USERNAMES_TO_SKIP.includes(user.username)) {
       continue
     }
-    if (recastCounts[user.address] >= MAX_RECAST_PER_USER) {
+    if (recastCounts[user.fid] >= MAX_RECAST_PER_USER) {
```

Comment:
We need to add `fid` to supabase query since we are now rely on the field. Currently we only query `username, address`. We can also remove `address` since we don't use it any more.

Reply:
done